tutorial.md

```diff
@@ -87,6 +107,22 @@ use the tags multiple times if you have multiple copyright holders or licenses.
 In the example project, you also edit `Makefile` and `README.md` using this
 header information, but of course with corresponding comment syntax.
 
+{{< box-tool >}}
+
+The `reuse addheader` command helps with adding licensing and copyright
+information to your files. For the task above, the following command
+would do the job:
+
+```bash
+reuse addheader --year="2019" --copyright="Jane Doe <jane@example.com>" --license="GPL-3.0-or-later" src/main.c Makefile README.md
```

Comment:
`--year` is superfluous if 2019 is the current year—the current year is added by default. Maybe it's good to exclude `--year` from this thing, because the line is already long as-is?

Reply:
I was unsure about this as well, because I wanted the result to be the same as in the tutorial. But you are right, this probably creates more confusion than it solves. Will fix